src/ExportMap.js

```diff
@@ -781,12 +782,29 @@ export function recursivePatternCapture(pattern, callback) {
   }
 }
 
+let parserOptionsHash = '';
+let prevParserOptions = null;
+let settingsHash = '';
+let prevSettings = null;
 /**
  * don't hold full context object in memory, just grab what we need.
+ * also calculate a cashKey, where parts of the cacheKey hash are memoized
```

Comment:
Haha. `cashKey`. I love it.

```suggestion
 * also calculate a cacheKey, where parts of the cacheKey hash are memoized
```